README.md

```diff
@@ -1,7 +1,7 @@
 # pixi-shadows
 
 Pixi-shadows allows you to add shadows and lights to your pixi stage.
-The lights themselves are very simple, and this plugin purely focuses on the shadows. If you are interested in high quality lighting, you can easily combine this plugin with the [pixi-lights](https://github.com/pixijs/pixi-lights) plugin. Check out the main demo [here](https://tarvk.github.io/pixi-shadows/build/demos/basic/).
+The lights themselves are very simple, and this plugin purely focuses on the shadows. If you are interested in high quality lighting, you can easily combine this plugin with the [pixi-lights](https://github.com/pixijs/pixi-lights) plugin. Check out the main demo [here](https://tarvk.github.io/pixi-shadows/examples/basic/).
 
 ![Teaser image](https://github.com/TarVK/pixi-shadows/raw/master/teaser.png)
```

Comment:
I believe you removed this image? We should probably put it back, or have some replacement. 
Also we can use normal relative linking, I think I may have used this absolute linking because I also built the readme to a html page in the past.

Reply:
yes! good catch